### PR TITLE
Update shipyard setting docs

### PR DESCRIPTION
**Pull request by MaxBab**
Add "gateways" parameter option to settings file of shipyrad. According to the following patch.
https://github.com/submariner-io/shipyard/pull/944

Signed-off-by: Maxim Babushkin <mbabushk@redhat.com>

### CODE REVIEW

File: src/content/development/shipyard/settings/_index.en.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ The possible settings are:
     and **`ovn`**.
   * **`nodes`**: A space separated list of nodes to deploy, supported types are **`control-plane`** and **`worker`**.
   * **`submariner`**: If Submariner should be deployed, set to **`true`**. Otherwise, leave unset (or set to **`false`** explicitly).
+  * **`gateways`**: Number of gateway nodes that needs to be deployed globally or on per cluster level.
```
Comment (skitt) [inline src/content/development/shipyard/settings/_index.en.md]: This is in the “Global and/or cluster specific” section, so “globally or on per cluster level” is specified by the section:

```suggestion
  * **`gateways`**: Number of gateway nodes to deploy.
```

File: src/content/development/shipyard/settings/_index.en.md [hunk 2]
```diff
@@ -62,3 +63,17 @@ clusters:
     submariner: false
     nodes: control-plane
 ```
+
+The following settings file deploys two clusters.
+The second cluster will deploy one control node and three worker nodes and 2 nodes will be labeled as gateway nodes.
```
Comment (skitt) [inline src/content/development/shipyard/settings/_index.en.md]:
```suggestion
The second cluster will deploy one control node and three worker nodes and two nodes will be labeled as gateway nodes.
```

File: src/content/development/shipyard/settings/_index.en.md [hunk 3]
```diff
@@ -62,3 +63,17 @@ clusters:
...
+As no **`gateways`** specified on global and cluster1 level, cluster1 will get a default number of 1 gateway node.
```
Comment (skitt) [inline src/content/development/shipyard/settings/_index.en.md]:
```suggestion
As no **`gateways`** setting is specified either globally or for the first cluster specifically, the first cluster will get the default single gateway node.
```

**Review by skitt — CHANGES_REQUESTED** [review]

File: src/content/development/shipyard/settings/_index.en.md (hunk 1 above)
Comment (MaxBab) [reply src/content/development/shipyard/settings/_index.en.md]: Done

File: src/content/development/shipyard/settings/_index.en.md (hunk 2 above)
Comment (MaxBab) [reply src/content/development/shipyard/settings/_index.en.md]: Done

File: src/content/development/shipyard/settings/_index.en.md (hunk 3 above)
Comment (MaxBab) [reply src/content/development/shipyard/settings/_index.en.md]: Done

File: src/content/development/shipyard/settings/_index.en.md [hunk 4]
```diff
@@ -62,3 +63,18 @@ clusters:
     submariner: false
     nodes: control-plane
 ```
+
+The following settings file deploys two clusters.
+The second cluster will deploy one control node and three worker nodes and two nodes will be labeled as gateway nodes.
```
Comment (mkolesnik) [inline src/content/development/shipyard/settings/_index.en.md]:
```suggestion
The second cluster will be deployed with one control node and three worker nodes, with two of the nodes labeled as gateway nodes.
```

File: src/content/development/shipyard/settings/_index.en.md [hunk 5]
```diff
@@ -62,3 +63,18 @@ clusters:
...
+As no **`gateways`** setting is specified either globally or for the first cluster specifically,
+the first cluster will get the default single gateway node.
```
Comment (mkolesnik) [inline src/content/development/shipyard/settings/_index.en.md]:
```suggestion
the first cluster will get have a single gateway node by default.
```

File: src/content/development/shipyard/settings/_index.en.md [hunk 6]
```diff
@@ -62,3 +63,18 @@ clusters:
     submariner: false
     nodes: control-plane
 ```
+
+The following settings file deploys two clusters.
+The second cluster will be deployed with one control node and three worker nodes, with two of the nodes labeled as gateway nodes.
+As no **`gateways`** setting is specified either globally or for the first cluster specifically,
+the first cluster will get have a single gateway node by default.
```
Comment (mkolesnik) [inline src/content/development/shipyard/settings/_index.en.md]:
```suggestion
As no **`gateways`** setting is specified either globally or for the first cluster specifically,
the first cluster will get have a single gateway node by default.
The second cluster will be deployed with one control node and three worker nodes, with two of the nodes labeled as gateway nodes.
```

Somehow reading "the second cluster" as the first paragraph, and then "... the first cluster" as the second, feels wrong to me.
Ordering them makes it better IMO

**Review by mkolesnik — APPROVED** [review]

**Review by skitt — APPROVED** [review]

**Pull request merged**